automation/swift-components-docs/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "swift-components-docs"
+edition = "2021"
+version = "0.1.0"
+authors = ["Sync Team <sync-team@mozilla.com>"]
+license = "MPL-2.0"
+
+[lib]
+crate-type = ["staticlib"]
+
+[dependencies]
+rust-log-forwarder = { path = "../../components/support/rust-log-forwarder" }
+viaduct = { path = "../../components/viaduct" }
+viaduct-reqwest = { path = "../../components/support/viaduct-reqwest" }
+nimbus-sdk = { path = "../../components/nimbus" }
+crashtest = { path = "../../components/crashtest" }
+fxa-client = { path = "../../components/fxa-client" }
+logins = { path = "../../components/logins" }
+autofill = { path = "../../components/autofill" }
+push = { path = "../../components/push" }
+tabs = { path = "../../components/tabs" }
+places = {path = "../../components/places" }
+remote_settings = { path = "../../components/remote_settings" }
+suggest = { path = "../../components/suggest" }
+sync15 = {path = "../../components/sync15"}
+error-support = { path = "../../components/support/error" }
+sync_manager = { path = "../../components/sync_manager" }
+as-ohttp-client = { path = "../../components/as-ohttp-client" }
```

Comment:
Could you use a dependency on `megazord_ios` instead of this list?  That way the crate list will always be kept in sync.